docs/src/main/asciidoc/security-jdbc-guide.adoc

```diff
@@ -0,0 +1,318 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Using Security with Jdbc realm
+
+include::./attributes.adoc[]
+
+This guide demonstrates how your Quarkus application can use a database to store your user identities.
+
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+== Architecture
+
+In this example, we build a very simple microservice which offers three endpoints:
+
+* `/api/public`
+* `/api/users/me`
+* `/api/admin`
+
+The `/api/public` endpoint can be accessed anonymously.
+The `/api/admin` endpoint is also protected with RBAC (Role-Based Access Control) where only users granted with the `admin` role can access. At this endpoint, we use the `@RolesAllowed` annotation to declaratively enforce the access constraint.
+The `/api/users/me` endpoint is protected with RBAC (Role-Based Access Control) where only users granted with the `user` role can access. As a response, it returns a JSON document with details about the user.
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `using-elytron-security-jdbc` {quickstarts-tree-url}/using-elytron-security-jdbc[directory].
+
+== Creating the Maven Project
+
+First, we need a new project. Create a new project with the following command:
+
+[source, subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=using-elytron-security-jdbc \
+    -Dextensions="elytron-security-jdbc, jdbc-h2, resteasy"
+----
+
+[NOTE]
+====
+Don't forget to to add the database connector library of choice. Here we are using H2 as identity store.
+====
+
+This command generates a Maven project, importing the `elytron-security-jdbc` extension
+which is an [`wildfly-elytron-realm-jdbc`](https://docs.wildfly.org/17/WildFly_Elytron_Security.html#jdbc-security-realm) adapter for Quarkus applications.

```

Comment:
```suggestion
which is a [`wildfly-elytron-realm-jdbc`](https://docs.wildfly.org/17/WildFly_Elytron_Security.html#jdbc-security-realm) adapter for Quarkus applications.
```
+ you are using a Markdown link, not an Asciidoc one (should be `https://mylink/[my text]`), it won't work well.